universal-login-relayer/lib/services/WalletService.ts

```diff
@@ -17,13 +19,15 @@ class WalletService {
   private abi: Abi;
   private contractJSON: ContractJSON;
   private useInitData: boolean;
+  public pendingExecutions: { [index: string]: PendingExecution; };
```

Comment:
I think best way to type it is:
`pendingExecutions : Record<string, PendingExecution>`